rif/relay/architecture.md

```diff
@@ -122,15 +146,18 @@ Detailed documentation can be found [here](https://eips.ethereum.org/EIPS/eip-71
 The Relay Server receives sponsored transactions via HTTP.
 
 The server has only one Relay Manager address and at least one Relay Worker, and points to just one Relay Hub.
-
 When the Relay Server receives an HTTP Relay request, it creates an Envelope, wrapping the sponsored transaction, signs it using its Relay Worker address and then sends it to the Relay Hub contract.
 
 The server is a service daemon, running as an HTTP service. It advertises itself (through the Relay Hub) and waits for client requests.
 
+The Relay Server has mechanisms that try to avoid running out of balance in the workers. The Relay Manager keeps sending native cryptocurrency to the workers based on a specific minimum balance.
+
```

Comment:
We should include a section to describe the start/register flows here.

Reply:
updated